sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1500,31 +1534,35 @@ object functions {
   }
 
   /**
-   * Computes the cosine of the given value. Units in radians.
+   * @param e angle in radians
+   * @return cosine of the angle, as if computed by [[java.lang.Math#cos]]
```

Comment:
Seems this link is broken in Scaladoc. Let's just use `` `java.lang.Math.cos` `` in this file.

Reply:
Replaced all of them as suggested.